### PR TITLE
Capitalization

### PR DESCRIPTION
**Pull request by andynava**
I think there should also be a word between "security practices, LibreSSL". Maybe "security practices, like LibreSSL", but not sure.

### CODE REVIEW

**Comment by mairaw** [comment]
Thanks for fixing the capitalization already. And you're right, it does seem like something is missing there.
@nishanil @ardalis can you review the sentence to suggest a fix here?

**Comment by ardalis** [comment]
Yeah, that sentence is difficult. I'd suggest simplifying it:

```
Examples include 
[Static analysis tools](https://www.whitesourcesoftware.com/), [encryption libraries](https://www.libressl.org/), and [security practices](https://azure.microsoft.com/resources/videos/red-vs-blue-internal-security-penetration-testing-of-microsoft-azure/).
```

File: docs/architecture/cloud-native/azure-security.md
Comment (mairaw) [inline docs/architecture/cloud-native/azure-security.md]:
```suggestion
Azure has all the tools necessary to release a highly secure product. However, a chain is only as strong as its weakest link. If the applications deployed on top of Azure aren't developed with a proper security mindset and good security audits, then they become the weak link in the chain. There are many great static analysis tools, encryption libraries, and security practices that can be used to ensure that the software installed on Azure is as secure as Azure itself. Examples include [static analysis tools](https://www.whitesourcesoftware.com/), [encryption libraries](https://www.libressl.org/), and [security practices](https://azure.microsoft.com/resources/videos/red-vs-blue-internal-security-penetration-testing-of-microsoft-azure/). 
```

**Comment by mairaw** [comment]
Thank @ardalis. Created a suggestion based on your comment. @andresnava can you please apply that? Thanks.

**Review by mvelosop — APPROVED** [review]
Thanks @andresnava!

It looks like this is only missing a formal review approval, so here it goes 😊

Ready to merge when you're ready @mairaw.

**Pull request merged**
